src/Home.tsx

```diff
@@ -200,8 +200,8 @@ const Home = ({
 											{data.contract && data.contract[activeNet].address}
 										</Heading>
 										<Heading subtitle size={6}>
-											<span>{`Hash script: `}</span>
-											{data.contract && data.contract[activeNet].script_hash}
+											<span>{`Script hash: `}</span>
+											0x{data.contract && data.contract[activeNet].script_hash}
```

Comment:
BTW, maybe this breaks TypeScript, I don't know.

Reply:
The idea is to have something like https://dora.coz.io/contract/neo3/mainnet/0x2cafa46838e8b564468ebd868dcafdd99dce6221

Reply:
Yeah, it should be
```
{`0x${data.contract && data.contract[activeNet].script_hash}`}
```

Reply:
Fixed.